### ISSUE TITLE
Look up steam appid for non-steam shortcuts

### ISSUE THREAD

**Issue by davr** [boppreh/steamgrid#21]
I have a lot of games that I bought through other sources (eg GOG) and then added to steam. It would be nice if it could look up the steam appid for these games, and then download the official steam banner, instead of only google image search.

I don't have the time to learn Go right now, so I wrote an extremely hacky way to do this, which you can see here: https://github.com/davr/steamgrid/commit/aa135f6f46c5cb72ddf6be67bd4f8b3566165297

Basically, I downloaded a list of a mapping of all steam appids, then I grep for the game names in that list to find the appid, then when downloading an image, I check both the original Id from the steam profile, as well as the Id found from the grep. Only works if the shortcut name is the same as the name in steam though, but most of mine are, so it was handy.

**Comment by boppreh** [comment]
That's a great idea. I'm not sure keeping a list of steamids is practical (it'll be large and get outdated), but it could do a simple search inside Steam. I'll look into this later today.

**Comment by boppreh** [comment]
Turns out Google already prioritizes official images from Steam. For example: https://www.google.com.br/search?tbs=isz%3Aex%2Ciszw%3A460%2Ciszh%3A215&tbm=isch&num=5&q=The%20Witcher%203 The first result is straight from the Steam store.

Do you have any example where the search fails to pickup the official image (when there is one)? Otherwise knowing the official ID for a game is interesting, but I don't see any clear uses at the moment.

Also, if we ever need an official ID we could use Steam's search suggestion, as in http://store.steampowered.com/search/suggest?term=witcher+3&f=games

**Comment by boppreh** [comment]
Hi @davr 

I'm closing this issue due to lack of activity. Again, I haven't seen a case where the lack of a Steam ID is a problem for the image search. If you have one, please reopen the issue.
